stable_baselines/common/base_class.py

```diff
@@ -422,29 +429,42 @@ def load_parameters(self, load_path_or_dict, exact_match=True):
         self.sess.run(param_update_ops, feed_dict=feed_dict)
 
     @abstractmethod
-    def save(self, save_path):
+    def save(self, save_path, cloudpickle=False):
         """
         Save the current parameters to file
 
-        :param save_path: (str or file-like object) the save location
+        :param save_path: (str or file-like) The save location
+        :param cloudpickle: (bool) Use older cloudpickle format instead of zip-archives.
         """
         raise NotImplementedError()
 
     @classmethod
     @abstractmethod
-    def load(cls, load_path, env=None, **kwargs):
+    def load(cls, load_path, env=None, custom_objects={}, **kwargs):
```

Comment:
```suggestion
    def load(cls, load_path, env=None, custom_objects=None, **kwargs):
```

You should never use a mutable as default arg ;)
 (an example of weird things happening because of that: https://twitter.com/araffin2/status/1122442000345829378)

you do a check later:
```python
if custom_objects is None:
    custom_objects = {}
```

Reply:
Oh dear. I was wondering why Codacy was warning about this :D. Thanks for clearing this up!